src/styles/catalog/cards.scss

```
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 300ms ease-in-out;
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    color: $code-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+    margin-bottom: 24px;
+  }
```

reset default browser styles at normalize.scss. Check other cases as well
```suggestion
  &__code {
    margin-bottom: 16px;
    color: $code-color;
  }

  &__rating {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 24px;
  }
```